src/lib.rs

```diff
@@ -89,8 +82,15 @@
 //!     # Ok(())
 //! # }
 //! ```
-#![deny(missing_docs, unused_import_braces, unused_qualifications)]
-#![warn(trivial_casts, trivial_numeric_casts, unsafe_code, unstable_features)]
+#![deny(
+    missing_docs,
+    intra_doc_link_resolution_failure,
+    clippy::all,
+    unsafe_code,
+    unreachable_pub,
+    unused,
+)]
+#![cfg_attr(docsrs, feature(doc_cfg))]
```

Comment:
This is the key in this PR. `docsrs` failed to generate documentation because we ask it to generate annotations on required features and failed to add this back then.